build/conf/kylin-server-log4j.properties

```diff
@@ -38,6 +38,7 @@ log4j.rootLogger=INFO
 log4j.logger.org.apache.kylin=DEBUG,file
```

Comment:
why not INFO for default？

Reply:
When user met some exceptions, they can find more detail which maybe helpful for cause analysis .

Reply:
> When user met some exceptions, they can find more detail which maybe helpful for cause analysis .

If some logs are needed for default, they should use log.info .
DDEBUG  is only needed by  test or debug problem。